kedify-agent/values.yaml

```diff
@@ -31,6 +31,9 @@ agent:
     pullPolicy: IfNotPresent
   imagePullSecrets: []
 
+  # comma-separated list of namespaces where pod utilization metrics should be collected
+  # or if unset denotes all namespaces, the collecting itself needs to be also enabled by KubernetesMetricsScraping flag
+  metricsWatchNamespace: ""
```

Comment:
should we consider adding `{{ .Release.Namespace }}` by default?

Reply:
yep, that's the plan, basically same logic as in keda upstream helm chart

```
  {{- $namespaces := append (splitList "," .Values.agent.metricsWatchNamespace ) .Release.Namespace -}}
  {{- range $namespaces }}
  ...
```